plugins/README.md

```diff
@@ -72,6 +76,16 @@ exports.teardown = function(config) {};
  * @return Return values are ignored.
  */
 exports.postResults = function(config) {};
+
+/**
+ * Called after each test block completes.
+ *
+ * @param {Object} config The plugin configuration object.
+ *
+ * @return Object If an object is returned, it is merged with the Protractor
+ *     result object. May return a promise.
+ */
+exports.postTest = function(config) {};
```

Comment:
You're right that these terms are overloaded and there's not much agreement.  We already use the word `spec` in our config files.  Maybe the word `test` is the best after all.